README.md

```diff
@@ -28,51 +28,42 @@ You can experience the capabilities of Asgardeo Tomcat OIDC Agent by following t
  sections listed below.
 
   * [Prerequisites](#prerequisites)
-  * [Configuring Identity Server](#configuring-identity-server)
-  * [Configuring the sample](#configuring-the-sample)
-  * [Running the sample](#running-the-sample)
+  * [Create Application in Asgardeo](#1-create-application-in-asgardeo)
+  * [Running the sample apps](#2-running-the-sample-apps)
```

Comment:
Do we need the `#2`? 🤔
If it works, then we can keep it?

Reply:
Yes. Having the numbers does work. Therefore decided to keep the `#1` and `#2` to match the topics.